nerfstudio/model_components/ray_samplers.py

```diff
@@ -505,7 +506,7 @@ def __init__(
         num_proposal_samples_per_ray: Tuple[int] = (64,),
         num_nerf_samples_per_ray: int = 32,
         num_proposal_network_iterations: int = 2,
-        single_jitter: bool = True,
+        single_jitter: bool = False,
```

Comment:
Set to True since we are using pointwise samples.

Reply:
This is just a default param, so it should be fine.